src/components/List.tsx

```diff
@@ -0,0 +1,77 @@
+import CoinData from "../types/CoinData";
+import React from "react";
+import { Image, Text, View } from "react-native";
+import styled from "styled-components";
+
+const CryptoContainer = styled(View)`
+    flex-direction: row;
+    justify-content: space-between;
+		align-items: center;
+		margin: 0px 24px;
+    padding: 30px 0px;
+    border-bottom-width: 1px;
+    border-color: #e5e5e5;
+`;
+
+const CryptoInfo = styled(View)`
```

Comment:
You should create a file for the styles.  like this

components/
   ComponentName/
           index.tsx
           styles